adminSiteServer/adminRouter.tsx

```diff
@@ -139,33 +135,6 @@ adminRouter.post(
 
 adminRouter.get("/logout", logOut)
 
-getPlainRouteWithROTransaction(
-    adminRouter,
-    "/datasets/:datasetId.csv",
```

Comment:
I was going under the assumption that we wouldn't need this route any more, since I don't know what it would be useful for.
This is its own commit, so gonna be very easy to revert if it's still useful.

Reply:
We don't use it anywhere in ETL (we grab datasets directly from the catalog), so if we don't use it anywhere in owid-grapher, it's ok to delete it.

Reply:
Up till now, there was a button on the dataset page to download a dataset. I'll ask around to see if it's still useful to anyone.